README.md

```diff
@@ -220,6 +241,12 @@ I would like to thank  Microverse
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./MIT.md) licensed.
+
```

Comment:
- [ ] I think it could be better to change your `[MIT](./MIT.md)` path to `[MIT](./LICENSE)` because we are getting an error message. Kindly fix it 👍  

![Screenshot 2023-05-09 122453](https://github.com/BANCUNGUYE66/book-store/assets/57408419/59e525cd-57da-478a-867a-dd2334bdb907)

Reply:
hello @joel-kalema I have done to edite but I can't push, have you done anything on this project?